hooks/_common.sh

```diff
@@ -233,7 +234,13 @@ function common::get_cpu_num {
       return
     fi
 
-    echo $((millicpu / 1000))
+    cpu_period=$(< /sys/fs/cgroup/cpu/cpu.cfs_period_us)
+    cpu_num=$((cpu_quota / cpu_period))
+    if ((cpu_num < 1)); then
+      echo 1
+    else
+      echo $cpu_num
+    fi
```

Comment:
@daniel-sampliner Do you think that there's no need for `echo $((millicpu / 1000))` as e.g. a fallback if `/sys/fs/cgroup/cpu/cpu.cfs_period_us` does not exist (in case it may not exist at all)? 🤔

Reply:
Good question. I don't believe it's possible for `cpu.cfs_period_us` to not exist if `cpu.cfs_quota_us` does exist and is > 0. But I also don't know for certain.

It definitely can't hurt to fall back to something if `cpu.cfs_period_us` doesn't exist, but I don't think `millicpu / 1000` would be the right value. Perhaps like this to default to 1 CPU when `cpu.cfs_period_us` doesn't exist

```suggestion
    cpu_period=$(< /sys/fs/cgroup/cpu/cpu.cfs_period_us || echo -1)
    cpu_num=$((cpu_quota / cpu_period))
    [[ $cpu_num -lt 1 ]] && echo 1 || echo $cpu_num
```

Reply:
Yep, makes sense to me.
@MaxymVlasov WDYT?